moq-pub/src/main.rs

```diff
@@ -17,13 +17,15 @@ use media::*;
 mod cli;
 use cli::*;
 
+use uuid::Uuid;
+
 // TODO: clap complete
 
 #[tokio::main]
 async fn main() -> anyhow::Result<()> {
 	env_logger::init();
 
-	let config = Config::parse();
+	let mut config = Config::parse();
 
 	let mut media = Media::new(&config).await?;
```

Comment:
We need to move the uuid namespace generation logic above here so we don't end up with an empty namespace in the catalog.

Reply:
Done